docs/source/prereqs/2-architecture.rst

```diff
@@ -1,10 +1,10 @@
 .. _hq-architecture:
 
-Architecture Overview
+Architecture and Platform Overview
 =====================
 
 It is useful to have some context on what it is that you are hosting,
 when you are hosting CommCare HQ.
 
 A good place to find that context is the `Architecture Overview <https://commcare-hq.readthedocs.io/overview/architecture.html>`_
-in the CommCare HQ documentation.
+and the `Platform Overview <https://commcare-hq.readthedocs.io/overview/platform.html>`_ the CommCare HQ documentation.
```

Comment:
nit: There is no one way to do this, though my preference would be to not club architecture and platform links together.
Architecture is probably the most relevant link followed by the [data flow diagram](https://dimagi.atlassian.net/browse/SC-3138?focusedCommentId=293559) I shared in the ticket

So I would say we make this look something like the following:
```
Architecture and Platform Overview
<same as before>
Other useful links:
- [Platform overview](https://commcare-hq.readthedocs.io/overview/platform.html)

Data Flow Overview
https://commcare-hq.readthedocs.io/overview/architecture.html#data-flow-for-forms-and-cases

```

Reply:
I think its fine if we skip the platform overview and add that to the architecture overview page itself.
Its quite scary and I don't think it is really useful for hosters unless someone is doing a deep understanding

Reply:
How's [this](https://github.com/dimagi/commcare-cloud/pull/6239/commits/12080bc961e3f12025ec8c1b1fc4e2b3690bbe8a)?

Reply:
Cool